components/dash-core-components/tests/integration/graph/test_graph_varia.py

```diff
@@ -178,11 +175,19 @@ def render_content(click, prev_graph):
         return prev_graph
 
     dash_dcc.start_server(app)
-    button = dash_dcc.wait_for_element("#click")
-    button.click()
+    dash_dcc.wait_for_element(".js-plotly-plot")
+    assert dash_dcc.driver.execute_script(
+        """
+        return (document.querySelector('.js-plotly-plot').data || []).length === 1;
+    """
+    )
+
+    dash_dcc.wait_for_element("#click").click()
     time.sleep(2)  # Wait for graph to re-render
-    dash_dcc.percy_snapshot(
-        "render-empty-graph ({})".format("eager" if is_eager else "lazy")
+    assert dash_dcc.driver.execute_script(
+        """
+        return (document.querySelector('.js-plotly-plot').data || []).length === 0;
+    """
```

Comment:
This could be refactored in a function to avoid repetition.

Reply:
definitely. https://github.com/plotly/dash/pull/2069/commits/4d869c2cc693f88ff22c5e2e013bd9d83397ad3f